index.bs

```diff
@@ -874,7 +879,7 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 
 							Issue: It should be possible to read the data asynchronously from the system clipboard after the author calls getType, however, this set of steps implies that data will be provided at the time of read.
 						
-						1. The user agent, MUST NOT sanitize |representation|'s [=representation/data=], if it satisfies the below conditions:
+						1. The user agent, MAY NOT sanitize |representation|'s [=representation/data=], if it satisfies the below conditions:
```

Comment:
We only mention UAs may not sanitize under certain condition, but I think we should also cover the default sanitization behavior.

I feel the original text before https://github.com/w3c/clipboard-apis/pull/197 can better cover all UAs default sanitization behavior. Maybe we can use the original text and saying that UA might support `optional unsanitized data types` to return unsanitized data?

Reply:
Sounds good to me. Added the original text with optional `unsanitized data types` text. Thanks!